### ISSUE TITLE
Obtaining the coordinates of the positions analysed by paftools.js call 

### ISSUE THREAD

**Issue by cvargas88** [lh3/minimap2#711]
Hi everyone. I am using `paftools.js call` to call the variants from an assembly to reference mapping. I used the following lines:
`minimap2 -cx asm10 --cs reference.fa assembly.fa | sort -k6,6 -k8,8n | paftools.js call -f reference.fa -`

I would like to obtain the positions that were not analysed by `paftools.js call`. From the output report I can see that: 206260985 reference bases covered by exactly one contig. 
However, when I try to obtain these coordinates from the .paf file I see that there are 209,711,902 bases in the reference covered by one contig. Are these the only positions where the variants are identified? Is there an extra filtering being applied? How can I obtain the coordinates for the 206,260,985 bases in the report?

Thanks a lot!

**Comment by lh3** [comment]
> Is there an extra filtering being applied?

Yes. paftools.js call filters out short alignments. See the command line prompt.
